src/main/java/model/Position.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * 클래스 이름
+ * 위치
+ * 버전 정보
+ * 1.0
+ * 날짜
+ * 2020.11.01
+ * 저작권 주의
+ */
+
+package model;
+
+public class Position {
+    private int position;
```

Comment:
```suggestion
    private int position;
    
```
불변 객체에 대해 알고 계시나요 ? 
알고 계신다면 Position을 불변으로 만들어 보는것도 좋을것 같아요.